tests/test.messages.js

```diff
@@ -42,6 +44,19 @@ describe('Messages', function() {
     }
   });
 
+  it('should have a legacyCode property in every message', () => {
+    for (let message in messages) {
+      if (typeof messages[message] === 'object') {
+        var legacyCode = messages[message].legacyCode;
+        if ((legacyCode instanceof Array && legacyCode.length !== 3) ||
+            (!(legacyCode instanceof Array) && legacyCode !== null)) {
+          assert.fail(null, null, singleLineString`A valide legacyCode could
+            not be found for code: "${messages[message].code}". Should be
+            an Array with 3 values based on the amo-validator err_id or null.`);
```

Comment:
This is the canonical place to add that note about reasons why we'd use a `null` value.